kyo-prelude/shared/src/main/scala/kyo2/Kyo.scala

```diff
@@ -0,0 +1,136 @@
+package kyo2
+
+import kernel.<
+import kernel.Frame
+import kernel.Loop
+import scala.annotation.tailrec
+
+object Kyo:
```

Comment:
I've moved some methods to the `Kyo` companion object. I'll review how this plays with `kyo-combinators` when porting it to the new design.